### PR TITLE
(WIP) Add K8s on Carina guide

### PR DESCRIPTION
**Pull request by jamiehannaford**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-feb06d88e0/). :bow:

File: _concepts/2016-11-04-kubernetes-on-carina.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: Kubernetes on Carina
+author: Jamie Hannaford <jamie.hannaford@rackspace.com>
+date: 2016-11-04
+permalink: docs/tutorials/kubernetes-on-carina/
+description: An overview of how Kubernetes is installed and operates Carina hosts
+kubernetes-versions:
+  - 1.4.1
```
Comment (lblackstone) [inline _concepts/2016-11-04-kubernetes-on-carina.md]: We have 1.4.4 deployed right now.

File: _concepts/2016-11-04-kubernetes-on-carina.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- `kubelet`
+- `kube-proxy`
+
+## Cluster components
+
+Once the Master and Node instances have been configured, we then install a
+selection of Kubernetes pods in order to help you get started. These pods
+include:
+
+- Flannel
```
Comment (lblackstone) [inline _concepts/2016-11-04-kubernetes-on-carina.md]: Flannel is installed as a systemd unit, not managed by k8s.

File: _concepts/2016-11-04-kubernetes-on-carina.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- DNS server
+- Ingress controller
+- Private Docker registry
```
Comment (lblackstone) [inline _concepts/2016-11-04-kubernetes-on-carina.md]: We aren't currently creating a registry with the clusters.

**Comment by jamiehannaford** [comment]
Integrating this into our Kubernetes 101 guide. Closing in favour of #890

**Pull request closed without merging**